src/utils/authentication.js

```diff
@@ -13,15 +13,29 @@ const signupValidation = [
     .withMessage('Your password must be at least eight charachters'),
 ];
 
-const generateToken = (user) => jwt.sign(
-  {
-    _id: user._id,
-    email: user.email,
-    username: user.username,
-    role: user.role,
-  },
-  process.env.JWT_SECRET,
-);
+const generateToken = user =>
+  jwt.sign(
+    {
+      _id: user._id,
+      email: user.email,
+      username: user.username,
+      role: user.role,
+    },
+    process.env.JWT_SECRET,
+    { expiresIn: 60 },
+  );
+
+const generateRefreshToken = user =>
+  jwt.sign(
+    {
+      _id: user._id,
+      email: user.email,
+      username: user.username,
+      role: user.role,
+    },
+    process.env.JWT_SECRET,
```

Comment:
do we need to use here maybe other secret for refresh token?

Reply:
they can be the same as well